windows/uipriv_windows.hpp

```diff
@@ -1,4 +1,6 @@
 // 21 april 2016
+
```

Comment:
Please remove these blank lines (revert the file back to master) as well. Thanks!